docs/coding-style.md

```diff
@@ -105,8 +105,9 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
             // Action
         }
     ```
-* Expression body of arrow property should be on the same line, as the property declaration. It should be on next line only when S103 is violated.
-* Expression body of method should be on the same line only for trivial cases: literal or simple identifier. Member access, indexer, invocation and other complex structures should be on the next line.
+* When using an arrow property or an arrow method, the `=>` token must be on the same line as the declaration. Regarding the expression body:
+  * for properties: it should be on the same line as the property declaration. It should be on the following line only when it is too long and would trigger [S103](https://sonarsource.github.io/rspec/#/rspec/S103/csharp).
```

Comment:
If you make this a link:
* Consider making all occurrences a link in this file
* Consider using rules.sonarsource.com instead of our backend

I'd say we don't need a link for simplicity. IDE and SC will tell all contributors when S103 is violated.